src/scripts/choices.ts

```diff
@@ -1438,8 +1438,8 @@ class Choices implements Choices {
     const hasActiveDropdown = this.dropdown.isActive;
     const hasItems = this.itemList.hasChildren();
     const keyString = String.fromCharCode(keyCode);
-    const wasAlphaNumericChar = /[a-zA-Z0-9-_ ]/.test(keyString);
-
+    const wasAlphaNumericChar = /[a-zA-Z0-9-_ |À|¿|¼|º|Þ|Ü|Û|Ý|¿|¾|`]/.test(keyString);
```

Comment:
Instead of continually adding characters to this check, let's instead check to see if user has entered a printable character, using:
```javascript
const wasPrintableChar = /[^\x00-\x1F]/.test(keyString);
```

Reply:
Greate, Thanks.
@mtriff

Reply:
Yes, please do.

Reply:
Done 
@mtriff